--- forge/fml/fml.py.orig	Tue Nov 12 17:38:02 2013
+++ forge/fml/fml.py	Tue Dec 10 12:40:02 2013
@@ -1146,7 +1147,7 @@
     # This list should be in the format of the new launcher's version.json file
     # Under the entry 'libraries'
     lib_dir = os.path.join(mcp_dir, 'jars', 'libraries')
-    default_url = 'http://s3.amazonaws.com/Minecraft.Download/libraries'
+    default_url = 'https://libraries.minecraft.net'
     
     downloads = []
     failed = False
